_includes/global_fields.md

```diff
@@ -4,6 +4,7 @@ A set of common fields that appear on all resources is included first, for refer
 
 | Name          | Type      | Description
 |-----------    |-----------|-----------|--------------
+|id 			| string    | A string representing a local identifier for the resource on the local system.  It may be a string, integer, guid, etc.  This value MUST be suitable for use in a helper function such as add_tags, add_lists to identify the corresponding resource
```

Comment:
I guess one big pro is that, with this constraint in place, if you tell me you have a tag whose id is X, then I know exactly how to get that tag for you - it's https://samplesystem.org/api/v1/tags/X, or whatever the endpoint is.  If there's no such constraint, there's no way to find "the tag whose id is X", which is pretty weird.